otx/algorithms/detection/adapters/mmdet/configurer.py

```diff
@@ -219,6 +218,17 @@ def update_config(src, pipeline_options):
             cfg.data.train.org_type = cfg.data.train.type
             cfg.data.train.type = super_type
 
+        for subset in ["train", "val", "test", "unlabeled"]:
```

Comment:
Couldn't we move these parts inside of `patch_persistent_workers`? It seems that we could do.

Reply:
It can affect to all mmXX tasks. But I think that's not bad idea to set their default value as True. I'll update it.